src/panel/NoResultPanel.jsx

```diff
@@ -1,26 +1,14 @@
-/* globals _ */
 import React from 'react';
 import { Panel } from 'src/components/ui';
+import NoResultMessage from './NoResultMessage';
 
 const close = () => window.app.navigateTo('/');
 
-const NoResultPanel = ({ resetInput }) => {
-  const tryNewSearch = e => {
-    e.preventDefault();
-    close();
-    resetInput();
-  };
-
+const NoResultPanel = () => {
   return (
     <Panel close={close}>
       <div style={{ padding: '16px 16px 32px 16px' }}>
-        <p className="u-mb-xs u-text--smallTitle">{_('No results found.')}</p>
-        <p className="u-text--subtitle u-mb-l">
-          {_('Check the spelling of your search or add more details, such as city or country.')}
-        </p>
-        <a onClick={tryNewSearch} href="#">
```

Comment:
Is is ok to just remove this action before we have something else to replace it?
Maybe we should keep it the meantime (for this case only, not during the suggest).

Or, if it's ok to get rid of it already, it deserves some cleaning in `PanelManager` to remove the `resetInput` function passed to `NoResultPanel` (if you do that, rebase your PR first as `PanelManager` changed a lot recently)